document-store/src/main/java/org/hypertrace/core/documentstore/postgres/utils/PostgresUtils.java

```diff
@@ -303,7 +333,7 @@ public static List<String> splitNestedField(String nestedFieldName) {
   }
 
   public static boolean isEncodedNestedField(String fieldName) {
-    return fieldName.contains(DOT_STR) ? true : false;
+    return fieldName.contains(DOT_STR) || fieldName.contains(DOT) ? true : false;
```

Comment:
Like mongo, support alias with `.` as the hierarchy of fields. See the test case `testAggregateWithNestedFields` in `DocStoreQueryV1Test.java`